### ISSUE TITLE
Sonata shouldn't be the default and only required font

### ISSUE THREAD

**Issue by donbyrd** [AMNS/Nightingale#54]
Inasmuch as we have no rights to the Adobe Sonata PostScript font, it shouldn't be the default and only required font! -- Briard should be. Unfortunately, several places in the code _assume_ Sonata is available, so this isn't that easy to change.

**Comment by donbyrd** [comment]
To clarify the issue, Ngale currently supports three fonts: Sonata, Briard, and BlueNotz. BlueNotz is a jazz/pop style font, so not appropriate as Ngale's default. Briard looks very much like Sonata; BlueNotz doesn't -- it's a jazz/pop style, copyist-style font. It was pretty easy to add support for Briard and BlueNotz when that was done some years ago because they're Sonata compatible. Are there any other Sonata compatible fonts? Also, the legal (intellectual property rights) status of any such fonts other than Sonata isn't clear.

**Comment by chirgwin** [comment]
@donbyrd [Bravura](http://www.smufl.org/fonts/) is worth considering:
> The Bravura font family is made available under the SIL Open Font License, which means that the fonts are free to download, use, embed, redistribute with other software (including commercial software) or to create derivative versions.

**Comment by donbyrd** [comment]
Yes, it'd be worth considering, except for one thing: It's not Sonata compatible. But that may not be as much of a problem as I've been thinking; as long as each of the Sonata glyphs we care about has a one-glyph equivalent in Bravura, it might just be a matter of using a different encoding with Bravura!
